dev-libs/libsigc++/libsigc++3-2.99.10.recipe

```diff
@@ -47,7 +47,10 @@ BUILD_PREREQUIRES="
 	cmd:perl
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:xsltproc$secondaryArchSuffix
-"
+	"
+
+defineDebugInfoPackage libsigc++$secondaryArchSuffix \
+	$libDir/libsigc-3.0.so.0.0.0
```

Comment:
```diff
defineDebugInfoPackage libsigc++$secondaryArchSuffix \
-	$libDir/libsigc-3.0.so.0.0.0
+	$libDir/libsigc-3.0.so.$libVersionCompat
```

BTW, you can add this after `INSTALL()`, because there is a testsuite :)
```

TEST()
{
	make check
}
```
FWIW, all tests are OK on x86_64 and x86 (primary and secondary):
```
============================================================================
Testsuite summary for libsigc++ 2.99.10
============================================================================
# TOTAL: 38
# PASS:  38
```